vaxrank/report.py

```diff
@@ -71,8 +71,8 @@ def ascii_report_from_ranked_vaccine_peptides(
         for j, vaccine_peptide in enumerate(vaccine_peptides):
             mutant_protein_fragment = vaccine_peptide.mutant_protein_fragment
             amino_acids = mutant_protein_fragment.amino_acids
-            mutation_start = mutant_protein_fragment.mutant_amino_acid_start_offset
```

Comment:
The "underlined" AA was one too far to the left actually. So the new output becomes, from the test script:

1) chr9 g.82927102G>T (Phip)
	Top score: 3.69
	Predicted effect: Substitution(variant=chr9 g.82927102G>T, transcript_name=Phip-003, transcript_id=ENSMUST00000034787, effect_description=p.T441N)
	Reads supporting variant allele: 17
	Reads supporting reference allele: 39
	Reads supporting other alleles: 0
	Vaccine Peptides:
		i. DNTVIHAV_N_NMTLKVWNSYTGQL (score = 3.69)
		   - Length: 23
		   - Expression score: 4.12
		   - Mutant epitope score: 0.90
		   - Wildtype epitope score: 1.03
		   - Reads fully spanning cDNA sequence(s): 2
		   - Mutant amino acids: 1
		   - Mutation distance from edge: 7
		   - Predicted mutant epitopes:
			 * HAVNNMTLKV 202.58 (H-2-Db)
			 * VIHAVNNM 714.63 (H-2-Kb)
			 * HAVNNMTL 1077.76 (H-2-Db)
			 * IHAVNNMTLKV 1968.09 (H-2-Db)